Vin205/Enyanjyoti
# Added a profile card Component

This pull request introduces a new ProfileCard component for the user dashboard. The ProfileCard displays the user's name, role, and includes a button to edit profile information.

Key Changes

- Created a new ProfileCard.js component under the components/Common directory.
- Integrated the ProfileCard into the Server.js
- Added state management to handle the "Edit Profile" button.
- Created a ProfileForm.js component to allow users to update their profile information.

How to Test

- Navigate to the dashboard.
- Verify that the ProfileCard displays user details.
- Click the "Edit Profile" button to see the profile editing form.
- Update the name field and save changes.
